### DISCUSSION TITLE
Quasar build command doesn't exit after building

### DISCUSSION THREAD

**Discussion by lopugit** [quasarframework/quasar#13137] [CLI - PWA mode] [answered]
I'm trying to deploy a quasar project to netlify, but it does this locally too. When I run quasar build it just finishes building and displays the tips messages, but doesn't exit the process. So the netlify deployment can't complete

I'm using quasar-cli 0.17.20 in an old project, any reason why the build command wouldn't exit?
<img width="1314" alt="Screen Shot 2022-04-10 at 8 36 51 pm" src="https://user-images.githubusercontent.com/13629190/162615200-4f84b254-a5db-4c6f-8a40-42fa4094ae87.png">

**Comment by nevesvox** [comment]
any news on this topic?

**Reply by rstoenescu** [reply, marked as answer]
Our support for v0.X ended many years ago. Unfortunately, those versions will not receive any fixes.

**Comment by h3** [comment]
A bit late to the party, but for what it's worth, I just had the same issue trying to "dockerize" an old web application.

It initially used node 12, but I upgraded to 16 in dev which worked fine until I tried to build for production.

Downgraded to node 14 and the issue went away.
